tests/test_ctl.py

```diff
@@ -402,9 +402,19 @@ def test_get_all_members(self):
     @patch('patroni.ctl.get_dcs')
     def test_members(self, mock_get_dcs):
         mock_get_dcs.return_value.get_cluster = get_cluster_initialized_with_leader
+
         result = self.runner.invoke(ctl, ['list'])
         assert '127.0.0.1' in result.output
         assert result.exit_code == 0
+        assert 'Citus cluster: alpha -' in result.output
```

Comment:
Citus case was actually working fine. What was broken - classic Patroni clusters.
Could we add a test for it?

Reply:
but I have added a test for a classic Patroni setup...